CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- Data structure for invalid options in events was changed from hashtable to
+list.
+
```

Comment:
The main reason for this PR is the bug causing that libreport asks for 'password' before it asks for 'login'. How about to say this instead of ...

@juliusmilan can you please help him to write a more informative message - I'm not going to do that intentionally - I'm too lazy :)